bin/sqlx-prepare

```diff
@@ -10,7 +10,7 @@ RELATIVE_PATH=$(realpath --relative-to="$(pwd)" "$FOLDER_PATH")
 
 run_command cd "$RELATIVE_PATH" || exit
 
-run_command cargo sqlx prepare --database-url postgres://headless-lms@localhost:54328/headless_lms_dev -- "$@" --lib
+run_command cargo sqlx prepare -- "$@" --lib
```

Comment:
This breaks the command for those that don't have the `.env` file set up in the headless-lms folder.

If the database url argumentl does not work on your computer, I think it can be resolved with updating the sqlx cli: `cargo install sqlx-cli --force`